utils/land-patterns.stanza

```diff
@@ -1034,3 +1089,154 @@ public pcb-landpattern smd-testpoint-pkg (d:Double) :
   layer(Courtyard(Top)) = Circle(d / 2.0)
   layer(Silkscreen("f-silk", Top)) = Circle(d / 2.0)
   ref-label()
+
+public defn make-two-pin-chip-landpattern (length:Toleranced,
```

Comment:
Please add a comment describing what this function is supposed to do. Then we can judge whether or not the API is appropriate. 

For example: Is this generator responsible for drawing the polarity marker when it is necessary? 

Currently it seems the answer is: It depends. 
1) When modeling a non-polarized component, this generator will handle the polarity marker appropriately. (By doing nothing.)
2) When modeling a polarized component, with pins a and c, this generator will handle the polarity marker appropriately. (By drawing it.)
3) When modeling a polarized component, with pins p[0] and p[1], (e.g. CAPT3216X160) then it is the caller's responsibility to pass `false` for the `polarized?` argument, and then manually call `draw-polarity-marker` themselves.

Point 3 when written out, makes it seem like this would be a tricky API to use.